### DISCUSSION TITLE
CAN dependencies

### DISCUSSION THREAD

**Discussion by AllonisDavid** [spbrogan/rvc2mqtt#100] [General]
Hi, Can you provide more detail on installing the CAN Waveshare dependencies?  Like which sections are used in the Waveshare doc that was referenced?  Or better, just copy / paste the pertinent information into the hardware doc for installation?  Thanx!!!

**Comment by spbrogan** [comment]
I updated the hardware directions at https://github.com/spbrogan/rvc2mqtt/blob/main/docs/hardware.md.  For Debian buster and bullseye I have not had to load any additional kernel mods so there is nothing needed for that.  You do still need to update the boot config to enable the SPI bus with the right parameters and leverage the network services to enable can0.  

Let me know if you have a different environment or process.  As with all things Linux there are probably 20 ways to do it. :)
